src/inference/src/system_conf.cpp

```diff
@@ -181,10 +186,9 @@ void set_cpu_used(const std::vector<int>& cpu_ids, const int used) {}
 
 #else
 
-static CPU cpu;
-
 #    ifndef _WIN32
 int get_number_of_cpu_cores(bool bigCoresOnly) {
+    CPU& cpu = cpu_info();
```

Comment:
please double-check its read/write concurrency protection, make sure there is no concurrency risk.

Reply:
Thanks for your remainder. Writable variables are _cpu_mapping_table[i][CPU_MAP_CORE_TYPE] and _proc_type_table, and add lock before reading and writing these variables.